Firestore/Source/Model/FSTFieldValue.mm

```diff
@@ -497,6 +509,10 @@ - (FSTTypeOrder)typeOrder {
   return FSTTypeOrderTimestamp;
 }
 
+- (id)value {
+  return self.localWriteTime;
```

Comment:
Hrm...  This makes me nervous because the default behavior is FSTServerTimestampBehaviorNone which returns NSNull...  So it seems like this should return NSNull or have a comment explaining what's going on.

Reply:
Hmm, I'm not sure. My understanding was that `value` should return some sort of a canonical representation of the object, which is independent from user-side settings (including the default behavior). So it seemed that `NSNull` is less appropriate - it seems to be just a way to drop the value. Another downside, in my view, is that the default behavior is now specified in two different places.

I don't feel strongly, though - feel free to push back.

Reply:
Honestly `value` shouldn't exist and I'm assuming this code isn't exercised since (I believe) you changed the value and it didn't break anything. And I'm not sure there is a good "canonical representation" for a server timestamp.  So I'd rather continue returning NSNull.

Keep in mind that on Android / JS, value and valueWithOptions are overloads of a single method, so I think it'd be strange to say that if you call it with no parameters it's the "canonical representation" but if you call it with an options param then it's the "user representation."

Reply:
Done.